pkg/controllers/provisioning/binpacking/packable.go

```diff
@@ -39,10 +41,36 @@ type Result struct {
 	unpacked []*v1.Pod
 }
 
+type Packables []*Packable
```

Comment:
I find this sort implementation pattern to be vastly inferior to https://pkg.go.dev/sort#Slice. You can separate out the comparator as necessary.

Reply:
ah you're right, I'll refactor those sorts.

Reply:
Did you forget to push?

Reply:
no? I might have done an intermediate push when you looked, should be okay now.